pkg/services/mssqlfg/database_pair_fe_primary_deprovision.go

```diff
@@ -0,0 +1,154 @@
+package mssqlfg
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Azure/open-service-broker-azure/pkg/service"
+)
+
+func (d *databasePairFePrimaryManager) GetDeprovisioner(
+	service.Plan,
+) (service.Deprovisioner, error) {
+	return service.NewDeprovisioner(
```

Comment:
Seems something is not write here, there are two `deletePriARMDeployment`

Reply:
Should be a mistake when rebasing. Thanks for the catch!